CHANGELOG.md

```diff
@@ -97,6 +100,8 @@ https://docs.docker.com/engine/deprecated/ where you can find the target removal
 + Add API support for templated secrets and configs [moby/moby#33702](https://github.com/moby/moby/pull/33702) and [moby/moby#36366](https://github.com/moby/moby/pull/36366)
 * Use rslave propagation for mounts from daemon root [moby/moby#36055](https://github.com/moby/moby/pull/36055)
 + Add /proc/keys to masked paths [moby/moby#36368](https://github.com/moby/moby/pull/36368)
```

Comment:
This is a security thing, if we have a security section already, it could be there (runtime is fine otherwise)